client/me/purchases/list/index.jsx

```diff
@@ -13,6 +13,9 @@ import Main from 'components/main';
 import MeSidebarNavigation from 'me/sidebar-navigation';
 import PurchasesHeader from './header';
 import PurchasesSite from './site';
+import sitesFactory from 'lib/sites-list';
```

Comment:
Instead of importing `SitesList` in `PurchasesList`, we should add it to `PurchasesData`.